src/nni_manager/common/utils.ts

```diff
@@ -268,5 +268,13 @@ function getIPV4Address(): string {
     throw Error('getIPV4Address() failed because no valid IPv4 address found.')
 }
 
-export { generateParamFileName, getMsgDispatcherCommand, getLogDir, getExperimentRootDir, 
+function getRemoteTmpDir(os_spec: string): string {
```

Comment:
use camel naming convention, like osSpec, but I suggest to rename to osType which is a little bit more clear